src/component/feature/variant/update-variant-component.jsx

```diff
@@ -80,13 +81,44 @@ class UpdateVariantComponent extends Component {
         </table>
     );
 
+    renderStickiness = variants => {
+        const { updateStickiness, stickinessOptions } = this.props;
+
+        if (!variants || variants.length < 2) {
+            return null;
+        }
+
+        const value = variants[0].stickiness || 'default';
+        const options = stickinessOptions.map(c => ({ key: c, label: c }));
+
+        // guard on stickiness being disabled for context field.
+        if (!stickinessOptions.includes(value)) {
+            options.push({ key: value, label: value });
+        }
+
+        const onChange = event => updateStickiness(event.target.value);
+
+        return (
+            <section style={{ paddingTop: '16px' }}>
+                <MySelect label="Stickiness" options={options} value={value} onChange={onChange} />
+                &nbsp;&nbsp;
+                <small>
+                    By overriding the stickiness you may control how users are distributed across variants.{' '}
```

Comment:
Suggestion: change "users" to "traffic" as this will no longer be sticky on user data.

Reply:
"By overriding the stickiness you can control which parameter you want to be used in order to ensure consistent traffic to a given feature." ?

Reply:
thanks, better!